website/integrations/services/awx-tower/index.md

```diff
@@ -77,4 +71,4 @@ In the `SAML Enabled Identity Providers` paste the following configuration:
 }
 ```
 
-`x509cert` is the certificate configured in authentik. Remove the `--BEGIN CERTIFICATE--` and `--END CERTIFICATE--` headers, then enter the cert as one non-breaking string.
+`x509cert` is the certificate configured in authentik. Remove the `--BEGIN CERTIFICATE--` and `--END CERTIFICATE--` headers, and enter the certificate as a single continuous string.
```

Comment:
```suggestion
`x509cert` is the certificate configured in authentik. Remove the `--BEGIN CERTIFICATE--` and `--END CERTIFICATE--` headers, and then enter the certificate as a single continuous string.
```

Reply:
Both my Editor at BMC and at IBM drilled the "and then" phrase into my head. ;-)